libs/base/ui/structure/collapsible/src/collapsible.component.ts

```diff
@@ -44,6 +55,16 @@ export class CollapsibleComponent
 
   protected onToggle(): void {
     this.open = this.details?.open;
+    if (this.isManuallyOpened && this.open) {
+      if (featureVersion >= '1.2') {
+        this.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
+      }
+    }
+    this.isManuallyOpened = false;
+  }
+
+  protected onClick(): void {
```

Comment:
A general concern, we could discuss during catchup. We do not have a strategy to prevent this. I cannot recall an industry standard to prepare for this case. I can imagine we move way from the binding we do in our handler names. In this case we could make it `onOpen`, but not too sure.

Reply:
Yeah, understand and probably worth to discuss and think how to deal with such cases.